lib/tests/streamlit/runtime/scriptrunner/script_runner_test.py

```diff
@@ -76,6 +79,16 @@ def _is_control_event(event: ScriptRunnerEvent) -> bool:
 
 @patch("streamlit.source_util._cached_pages", new=None)
 class ScriptRunnerTest(AsyncTestCase):
+    def setUp(self) -> None:
+        super().setUp()
+        media_file_manager._media_file_manager = MediaFileManager(
+            MemoryMediaFileStorage("/mock/media")
+        )
```

Comment:
As above, lots of tests rely on the MediaFileManager singleton existing, so we create it in this test subclass.